tutorials/migrating/upgrading_to_godot_4.3.rst

```diff
@@ -0,0 +1,350 @@
+Method ``add_control_to_dock`` adds a new ``shortcut`` optional parameter                                                 |✔️|                 |✔️ with compat|      |✔️|                  `GH-88081`_
+**EditorSceneFormatImporterFBX**
+Type renamed to ``EditorSceneFormatImporterFBX2GLTF``                                                                     |❌|                 |❌|                  |❌|                  `GH-81746`_
+========================================================================================================================  ===================  ====================  ====================  ===========
+
+Behavior changes
+----------------
+
+In 4.3 some behavior changes have been introduced, which might require you to adjust your project.
+
+Core
+^^^^
+
+.. note::
+
+    Binary serialization was modified to fix some issues with the serialization of scripted Objects and typed Arrays (`GH-78219`_).
+    This breaks compat with script encoding/decoding.
+
+.. note::
+
+    ``PackedByteArray`` is now encoded as a base64 string, this breaks compatibility in resources that contain them
+    making them no longer usable in older versions of the engine (`GH-89186`_).
+
+.. note::
+
+    In C#, the ``Transform3D.InterpolateWith`` implementation was fixed to use the right order of operations, applying the rotation before the scale (`GH-89843`_).
+
+.. note::
+
+    In C#, the ``Aabb.GetSupport`` implementation was fixed to properly return the support vector (`GH-88919`_).
+
+.. note::
+
+    In C#, the Variant types' ``ToString`` implementation now defaults to using the ``InvariantCulture`` (`GH-89547`_)
+    which means ``Vector2(1.2, 3.4)`` is formatted using ``.`` as the decimal separator independently of the language
+    of the operating system that the program is running on.
+
+Animation
+^^^^^^^^^
+
+.. note::
+
+    ``AnimationMixer`` replaced its Capture mode with a new Capture feature that works much better than the old one,
+    this replaces the existing cache (`GH-86715`_).
+
+.. note::
+
+    ``AnimationNode`` has a reworked process for retrieving the semantic time info. This ensures that time-related
+    behavior works as expected, but changes the blending behavior. Implementors of the ``_process`` virtual method
+    should also note that this method is now deprecated and will be replaced by a new one in the future (`GH-87171`_).
+
+GUI nodes
+^^^^^^^^^
+
+.. note::
+
+    The default font outline color was changed from white to black. (`GH-54641`_).
+
+.. note::
+
+    The ``auto_translate`` property is deprecated in favor of the ``auto_translate_mode`` property which is now in ``Node`` (`GH-87530`_).
+    Existing nodes that with ``auto_translate`` enabled may no longer be translated if their parent does not enable ``auto_translate``.
```

Comment:
This looks weirdly worded to me.

Reply:
Oh, you are right. I have no idea what I was trying to say, I might have been distracted when I wrote it :smile: 

Perhaps I was trying to explain the new `AUTO_TRANSLATE_MODE_INHERIT` default, paraphrasing the second point raised in https://github.com/godotengine/godot/pull/87530#issuecomment-1929442975.